src/main/conf/ds-cumulus-export-default-mapping.yml

```diff
@@ -96,10 +99,48 @@ digisam:
           replacement: "https://kb-images.kb.dk/DAMJP2/$1/"
           lineBreakIsMulti: false
 
-        - source: "Horizontal Pixels"
+        - source: "Horizontal Pixels"  # COP: part of Dimension
           dest: "width_pixels"
           destType: "long"
 
-        - source: "Vertical Pixels"
+        - source: "Vertical Pixels"  # COP: part of Dimension
           dest: "height_pixels"
           destType: "long"
+
+        - source: "Record Name"  # COP: Id
+          dest: "record_name"
+
+        - source: "Materialebetegnelse"  # COP: Ressourcetype 1 (If missing: Generel materialebetegnelse below is shown)
+          dest: "material_1"
+
+        - source: "Generel materialebetegnelse"  # COP: Ressourcetype 2
+          dest: "material_2"
+
+        - source: "Opstilling"  # COP: Opstilling
+          dest: "archive" # Better tag needed
```

Comment:
Sample entries are
```
Billedsamlingen. Kistebilleder. Cl. 121a. Box
Billedsamlingen. Danske portrætter.
Billedsamlingen. Kistebilleder. Box
Billedsamlingen. Kistebilleder. Cl. 5. Box
```
Maybe `internal_location` or `archive_location`?

Reply:
archive_location

Reply:
rename to archive_location